src/main/java/net/sf/jabref/logic/search/rules/RegexBasedSearchRule.java

```diff
@@ -51,9 +52,13 @@ public boolean applyRule(String query, BibEntry bibEntry) {
 
         for (String field : bibEntry.getFieldNames()) {
             if (bibEntry.hasField(field)) {
-                String fieldContent = RegexBasedSearchRule.REMOVE_LATEX_COMMANDS
-                        .format(bibEntry.getFieldOptional(field).get());
-                String fieldContentNoBrackets = RegexBasedSearchRule.REMOVE_LATEX_COMMANDS.format(fieldContent);
+                Optional<String> fieldOptional = bibEntry.getFieldOptional(field);
```

Comment:
Merge with the previous line:

    Optional<String> fieldOptional = bibEntry.getFieldOptional(field);
    if (fieldOptional.isPresent()) {
       ... remaining code...

Then, you also don't need the check on line 56.